backend/dataall/modules/s3_datasets_shares/api/queries.py

```diff
@@ -72,10 +26,31 @@
     test_scope='Dataset',
 )
 
+listShareObjects = gql.QueryField(
+    name='listDatasetShareObjects',
+    resolver=list_dataset_share_objects,
+    args=[
+        gql.Argument(name='datasetUri', type=gql.NonNullableType(gql.String)),
+        gql.Argument(name='environmentUri', type=gql.String),
+        gql.Argument(name='page', type=gql.Integer),
+    ],
+    type=gql.Ref('ShareSearchResult'),
+)
```

Comment:
Is this query used anywhere - should we leave a comment to clean up

Reply:
No, it is legacy. It must have been used in the shares tab (but now we use the inbox/outbox as we do in the shares view) Removing it!